sdks/java/io/cassandra/src/main/java/org/apache/beam/sdk/io/cassandra/CassandraIO.java

```diff
@@ -375,9 +375,15 @@ private CassandraIO() {}
   static class CassandraSource<T> extends BoundedSource<T> {
     final Read<T> spec;
     final List<String> splitQueries;
+    final Long estimatedSize;
```

Comment:
Could you please add some comments about this field? Like, why and when the size is cached.

Reply:
done.